docs/docsite/rst/user_guide/windows_setup.rst

```diff
@@ -456,6 +456,10 @@ Ansible, select one of these three installation options:
 
 * Manually install the service, following the `install instructions <https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH>`_
   from Microsoft.
+  
+* Install the `openssh <https://chocolatey.org/packages/openssh>`__ package using Chocolatey::
```

Comment:
There's an extra underline character here:

```suggestion
* Install the `openssh <https://chocolatey.org/packages/openssh>`_ package using Chocolatey::
```
Accept this suggestion to remove it.

Reply:
Thanks. Fixed it. (The GitHub UI was rather confusing with the live syntax highlighting/rendering in this case.)